src/reaper_osara.cpp

```diff
@@ -4810,6 +4811,62 @@ void cmdInsertMarker(Command* command) {
 	outputMessage(format(translate("marker {} inserted"), number));
 }
 
+void cmdInsertProjectMarker(Command* command) {
+	cmdhInsertProjectMarker(command->gaccel.accel.cmd);
+}
+
+void cmdhInsertTakeMarker(int command) {
+	if(CountSelectedMediaItems(0)>0) {
+		vector<int> preMarkers(CountSelectedMediaItems(nullptr));
+		for(int i = 0; i < preMarkers.size(); ++ i) {
+			MediaItem* item = GetSelectedMediaItem(nullptr, i);
+			MediaItem_Take* take = GetActiveTake(item);
+			preMarkers[i] = GetNumTakeMarkers(take);
+		}
+		Main_OnCommand(command, 0);
+		int addedMarkers = 0;
+		for(int i = 0; i < preMarkers.size(); ++ i) {
+			MediaItem* item = GetSelectedMediaItem(nullptr, i);
+			MediaItem_Take* take = GetActiveTake(item);
+			if (preMarkers[i] < GetNumTakeMarkers(take)) {
+				++ addedMarkers;
+			}
+		}
+		if (addedMarkers == 0) {
+			return; // not inserted
+		} else {
+			// Translators: Reported when using REAPER's quick add take marker action. If more than one take marker is inserted, [] will be replaced with the number of markers. E.G. "2 take markers inserted".
+			outputMessage(format(
+				translate_plural("take marker inserted", "{} take markers inserted", addedMarkers), addedMarkers));
+			return;
```

Comment:
There's no need for a return if it's the last thing in the function and the function returns nothing.

```suggestion
```